operator/pkg/helmreconciler/prune.go

```diff
@@ -73,6 +73,7 @@ var (
 	// ClusterCPResources lists cluster scope resources types which should be deleted during uninstall command.
 	ClusterCPResources = []schema.GroupVersionKind{
 		{Group: "admissionregistration.k8s.io", Version: "v1", Kind: name.MutatingWebhookConfigurationStr},
+		{Group: "admissionregistration.k8s.io", Version: "v1", Kind: name.ValidatingWebhookConfigurationStr},
```

Comment:
Will this also remove the ones from `base`?

Reply:
The one in base gets `istio.io/rev` from the last revision installed, so if `istioctl uninstall`ing the last revision installed it will be removed... so maybe this is dangerous...